00-Login/README.md

```diff
@@ -0,0 +1,86 @@
+# Auth0 React Native Samples - Login
+
+The complete guide to get started with the [react-native-auth0](https://github.com/auth0/react-native-auth0) SDK is on the Auth0 docs link below.
+
+- [Login QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login)
+
+
+## How to run the sample app
+
+Clone the repository and install the dependencies:
+
+```bash
+yarn install
+```
+
+#### Additional step: iOS
+
+Change the directory into the `ios` folder. You must run the following command to install the SDK pod.
+
+```bash
+# cd ios/
+pod install
+```
+
+You should see the `A0Auth0` pod being installed and linked to the sample app.
+
+
+### Auth0 Credentials 
+
+Open the [Auth0 dashboard](https://manage.auth0.com/dashboard/). Select your existing **Native** type application or create a new one. From the top of the application settings page take the `Client ID` and `Domain` values and put them in the `app/auth0-credentials.js` file. This file is being used in the `app/index.js` file to instantiate the library. You can also change and do that manually this way:
+
+```js
+import Auth0 from 'react-native-auth0';
+
+const config = {
+  domain: 'lbalmaceda.auth0.com',
+  clientId: '4e29mEHknIb1iKesCyeleWJUcz6fVR0O'
+}
+
+const auth0 = new Auth0(config);
+```
+
+### Deep linking setup
+The methods required to handle the browser deep link are already in place. If you need to review them, check the full article linked on top of this document.
+
+The URL that the Browser App calls back with the authentication result is created using a combination of the following values.
+- package name / product bundle identifier; currently `com.auth0samples` on both platforms.
+- platform name
+- auth0 domain
+
+If you happen to change the first remember to take note of it to update the [Auth0 dashboard](https://manage.auth0.com/dashboard/) configuration later.
```

Comment:
What is a "dashboard configuration?" Do you mean the Application? Which field?

Reply:
I mean the "whitelisting url" section below, which I can ref/link actually. Let me do that.